alf/optimizers/optimizers.py

```diff
@@ -45,6 +109,11 @@ def __init__(self,
             clip_by_global_norm (bool): If True, use `tensor_utils.clip_by_global_norm`
                 to clip gradient. If False, use `tensor_utils.clip_by_norms` for
                 each grad.
+            parvi (string): if not ``None``, paramters with attribute
+                ``ensemble_group`` will be updated by particle-based vi algorithm
+                specified by ``parvi``, options are [``svgd``, ``gfsf``].
```

Comment:
Need to document the attributes of parameter so that parvi can work properly.

Reply:
Added.